src/Containers/Microsoft.NET.Build.Containers/KnownStrings.cs

```diff
@@ -40,6 +40,7 @@ public static class ErrorCodes
         public static readonly string CONTAINER2009 = nameof(CONTAINER2009);
         public static readonly string CONTAINER2010 = nameof(CONTAINER2010);
         public static readonly string CONTAINER2012 = nameof(CONTAINER2012);
+        public static readonly string CONTAINER2014 = nameof(CONTAINER2014);
```

Comment:
this code is not defined, is it needed?

Reply:
This is defined - it's the InvalidContainerImageName error that we throw in the validation, and I wanted a constant to refer to it by in the test I added.

Having said that, if we tweak the message to be more descriptive then a new code would be necessary.